lib/api/CFieldDataCategorizer.cc

```diff
@@ -492,7 +501,7 @@ bool CFieldDataCategorizer::handleControlMessage(const std::string& controlMessa
         break;
     case 'f':
         // Flush ID comes after the initial f
-        this->acknowledgeFlush(controlMessage.substr(1));
+        this->acknowledgeFlush(controlMessage.substr(1), lastHandler);
         break;
     default:
         LOG_WARN(<< "Ignoring unknown control message of length "
```

Comment:
This warning should be inside an `if (lastHandler)` test, because otherwise it will generate spam for all the anomaly detector control messages.